packages/react-core/src/components/DualListSelector/examples/DualListSelector.md

```diff
@@ -23,806 +24,86 @@ import PficonSortCommonAscIcon from '@patternfly/react-icons/dist/esm/icons/pfic
-              ]
-            },
-            { id: 'CO4', text: 'Chosen Option 4', isChecked: false, checkProps: { 'aria-label': 'Chosen Option 4' } }
-          ]
-        }
-      ],
-      availableOptions: [
-        {
-          id: 'F1',
-          text: 'Folder 1',
-          isChecked: false,
-          checkProps: { 'aria-label': 'Folder 1' },
-          hasBadge: true,
-          badgeProps: { isRead: true },
-          children: [
-            { id: 'O1', text: 'Option 1', isChecked: false, checkProps: { 'aria-label': 'Option 1' } },
-            {
-              id: 'F1A',
-              text: 'Folder 1A',
-              isChecked: false,
-              checkProps: { 'aria-label': 'Folder 1A' },
-              children: [
-                { id: 'O2', text: 'Option 2', isChecked: false, checkProps: { 'aria-label': 'Option 2' } },
-                { id: 'O3', text: 'Option 3', isChecked: false, checkProps: { 'aria-label': 'Option 3' } }
-              ]
-            },
-            { id: 'O4', text: 'Option 4', isChecked: false, checkProps: { 'aria-label': 'Option 4' } }
-          ]
-        },
-        { id: 'O5', text: 'Option 5', isChecked: false, checkProps: { 'aria-label': 'Option 5' } },
-        {
-          id: 'F2',
-          text: 'Folder 2',
-          isChecked: false,
-          checkProps: { 'aria-label': 'Folder 2' },
-          children: [
-            { id: 'O6', text: 'Option 6', isChecked: false, checkProps: { 'aria-label': 'Option 6' } },
-            { id: 'O7', text: 'Option 5', isChecked: false, checkProps: { 'aria-label': 'Option 5 duplicate' } }
-          ]
-        }
-      ]
-    };
-
-    this.onListChange = (newAvailableOptions, newChosenOptions) => {
-      this.setState({
-        availableOptions: newAvailableOptions,
-        chosenOptions: newChosenOptions
-      });
-    };
-  }
-
-  render() {
-    return (
-      <DualListSelector
-        availableOptions={this.state.availableOptions}
-        chosenOptions={this.state.chosenOptions}
-        onListChange={this.onListChange}
-        id="basicSelector"
-        isSearchable
-        isTree
-      />
-    );
-  }
-}
+```ts file="./DualListSelectorTree.tsx"
 ```
 
 ### Composable dual list selector
 
-For more flexibility, a Dual list selector can be built using sub components. When doing so, the intended component
-relationships are arranged as follows:
+For more flexibility, a dual list selector can be built using sub components. When doing so, the intended component relationships are arranged as follows:
 
 ```js noLive
 import React from 'react';
-import { DualListSelector, DualListSelectorPane, DualListSelectorList, DualListSelectorListItem, DualListSelectorControlsWrapper, DualListSelectorControl } from '@patternfly/react-core';
+import {
+  DualListSelector,
+  DualListSelectorPane,
+  DualListSelectorList,
+  DualListSelectorListItem,
+  DualListSelectorControlsWrapper,
+  DualListSelectorControl
+} from '@patternfly/react-core';
```

Comment:
Could you just remove these imports from the `noLive` region? They are clutter and not helping to demonstrate structure as was the intent of this documentation.